chapter_appendix_math/statistics_vn.md

```diff
@@ -760,22 +764,28 @@ We close this section by providing three tempting but false statements.
 An in-depth discussion of these points can be found in :cite:`Morey.Hoekstra.Rouder.ea.2016`.
 -->
 
-*dịch đoạn phía trên*
+Điều này nghe có vẻ tiểu tiết, nhưng lại có một ý nghĩa quan trọng trong việc diễn giải các kết quả.
+Cụ thể, chúng ta có thể thỏa mãn :eqref:`eq_confidence` bằng cách tạo ra các khoảng *gần như chắc chắn* là chúng không tồn tại giá trị thực, miễn là số lượng các khoảng này không nhiều.
+Chúng ta kết luận mục này bằng cách đưa ra ba giả thuyết nghe hợp lý nhưng lại không chính xác.
+Một sự thảo luận sâu hơn về những điểm dưới đây có thể tìm thấy ở  :cite:`Morey.Hoekstra.Rouder.ea.2016`.  
 
 <!--
 * **Fallacy 1**. Narrow confidence intervals mean we can estimate the parameter precisely.
 * **Fallacy 2**. The values inside the confidence interval are more likely to be the true value than those outside the interval.
 * **Fallacy 3**. The probability) that a particular observed $95\%$ confidence interval contains the true value is $95\%$.
 -->
 
-*dịch đoạn phía trên*
+* **Ngụy biện 1**. Khoảng tin cậy hẹp cho phép chúng ta dự đoán các giá trị một cách chính xác.
+* **Ngụy biện 2**. Các giá trị nằm trong khoảng tin cậy có nhiều khả năng là giá trị thực hơn là các giá trị nằm bên ngoài.  
+* **Ngụy biện 3**. Xác xuất mà một khoảng được quan sát với độ tin cậy $95\%$ chứa các giá trị thực là $95\%$.
 
 <!--
 Sufficed to say, confidence intervals are subtle objects.  H
 owever, if you keep the interpretation clear, they can be powerful tools.
 -->
 
-*dịch đoạn phía trên*
+Có thể nói, các khoảng tin cậy là những đối tượng khó đoán.
```

Comment:
```suggestion
Có thể nói, các khoảng tin cậy là những đối tượng khó ước lượng.
```

Reply:
```suggestion
Có thể nói, các khoảng tin cậy là những đối tượng khá tế nhị.
```
`subtle` chỗ này không phải là về cơ chế tính toán của nó khó hay dễ, mà ý chỉ các cách hiểu với diễn đạt.

Reply:
tế nhị là những tình tiết rất nhỏ, sâu kín, khó nói hoặc không thể nói ra được.

ở đây là dễ nhầm lẫn hơn là khó nói.